common-moonmage.lic

```diff
@@ -127,15 +127,46 @@ module DRCMM
   # Returns false if you're not holding a moon weapon, or you are but can't wear it.
   # https://elanthipedia.play.net/Shape_Moonblade
   def wear_moon_weapon?
-    moon_weapon_regex = /moon[\w]+/
-    moon_wear_messages = ["telekinetic", "already wearing", "You can't wear", "Wear what"]
+    moon_weapon_regex = /((black|red-hot|blue-white) moon[\w]+)/
+    moon_wear_messages = ["You're already", "You can't wear", "Wear what", "telekinetic"]
```

Comment:
Move "telekinetic" match to last because if you're already wearing a moon weapon and you have a second one in your hands and you try to wear it the game will respond with "You're already telekinetically supporting <moon weapon>". So the word "telekinetic" which is to match the success case was also matching the failure case.

Another consideration instead of matching on "telekinetic" for the success case is "You gently suspend" but I'm not experienced enough with moon weapons to know if that messaging ever changes due to weapon shape, strength of the spell, etc etc so I kept it as telekinetic.